addons/survey/views/survey_survey_views.xml

```diff
@@ -428,56 +428,47 @@
             <p>
                 No Survey Found
             </p><p>
-                Ready to test? Pick a sample or create one from scratch...
+                Pick a sample or <a action="action_load_sample_custom" class="o_survey_sample_container o_survey_load_sample">Start from scratch</a>.
             </p>
-            <div class="row mb-4">
-                <div class="col-sm-3 o_survey_sample_container" action="action_load_sample_survey">
-                    <div class="o_survey_sample_tile position-relative w-100 d-flex align-items-center justify-content-center m-auto">
-                        <img src="/survey/static/src/img/survey_sample_survey.png" class="img-fluid"/>
-                        <div class="o_survey_sample_tile_cover o_survey_load_sample w-100 h-100 position-absolute rounded-circle text-center text-white p-3">
-                            Gather feedbacks from your employees and customers
-                        </div>
-                    </div>
-                    <div class="my-3">
-                        <a class="o_survey_load_sample h3 text-center text-primary">Survey</a>
-                    </div>
-                    <a class="btn btn-outline-primary o_survey_load_sample"><span>Try It</span></a>
-                </div>
-                <div class="col-sm-3 o_survey_sample_container" action="action_load_sample_assessment">
-                    <div class="o_survey_sample_tile position-relative w-100 d-flex align-items-center justify-content-center m-auto">
-                        <img src="/survey/static/src/img/survey_sample_assessment.png" class="img-fluid"/>
-                        <div class="o_survey_sample_tile_cover o_survey_load_sample w-100 h-100 position-absolute rounded-circle text-center text-white p-3">
-                            Handle quiz &amp; certifications
+            <div class="row mb-4 w-100 px-5 justify-content-center">
+               <div class="col-md-4 col-lg-3 py-4 o_survey_sample_container o_survey_load_sample" action="action_load_sample_survey">
+                    <div class="survey_card card rounded p-3 d-flex align-items-stretch h-100">
+                        <div class="row m-0 w-100 h-100">
+                            <div class="d-flex col-lg-4 align-items-start justify-content-center display-3 p-3">
+                                <img src="/survey/static/src/img/survey_sample_survey.png" class="img-fluid"/>
+                            </div>
+                            <div class="col-lg-8 p-3 card-body d-flex flex-column align-items-start justify-content-start h-100">
```

Comment:
Same thing here, if I just keep "col-lg-8 p-3" and add "text-start", layout is seemingly the same and a lot more simple.

Even "card-body" doesn't seem to do much, which makes me wonder if we need all these "card-xxx" classes at all?